projects.html

```diff
@@ -21,6 +21,39 @@
   <script type="text/javascript">window.dataLayer = window.dataLayer || [];function gtag(){dataLayer.push(arguments);}gtag('js', new Date());gtag('config', 'UA-182992779-1', {'anonymize_ip': false});</script>
 </head>
 <body>
+  <div style="display:none;opacity:0" class="nzcsa_modal">
+    <div class="div-block-26">
+      <div data-w-id="2ad46b46-ed74-e8a7-2239-11306c07d122" class="close_button">X</div>
+      <h2 class="heading-214"><strong>NZCSA</strong> - NEW ZEALAND CHINESE STUDENT ASSOCIATION
+        WECHAT MINI-PROGRAM</h2><img src="images/NZCSA_Demo.png" loading="lazy" sizes="(max-width: 479px) 68vw, 100vw" alt="" class="image-25">
+      <p class="paragraph-16" style="margin-top: 20px;">THe NZCSA project is the event platform for the New Zealand Chinese Student Association memebrs to register their events in NZCSA. It allows the memebrs to sign in with their registered email account and use it as a proof for the event. This platform also support online transactions so that the memebrs can pay more easily.<br>‍<br>Checkout our live site on: <a href="https://nz.linkedin.com/in/meloguan" target="_blank">nzcsa.wdcc.co.nz<br></a><br>Project Manager: <a href="https://nzcsa.wdcc.co.nz" target="_blank">Melo Guan</a><br>Project memebrs: Kirsty, Zhiqing, Yameizhen, Tony, Garfield, Linkun, Alex &amp; Jack</p>
+    </div>
+    <div data-w-id="2af6eddc-429e-a520-d6c8-b14c9d5050f7" class="close"></div>
+  </div>
+  <div style="display:none;opacity:0" class="nzpmc_modal">
+    <div class="div-block-26">
+      <div data-w-id="abf03c44-051c-c0ba-be7f-5e229d2908a9" class="close_button">X</div>
+      <h2 class="heading-214"><strong>NZPMC</strong> - NEW ZEALAND PHYSICS & MATHS COMPETITION WEBSITE</h2><img src="images/NZPMC_Demo.png" loading="lazy" sizes="(max-width: 479px) 68vw, 100vw" alt="" class="image-25">
+      <p class="paragraph-16" style="margin-top: 20px;">The NZPMC Project start on March 2021 where the project mamanger takes on developing a website/platform for the Neew Zealand Physics and Mathematics Competitaion. It is built in Vue.Js.<br>‍<br>Checkout our live site on: <a href="https://nzpmc.wdcc.co.nz" target="_blank">nzpmc.wdcc.co.nz<br></a><br>Project Manager: <a href="https://nz.linkedin.com/in/will-zhang-uoa" target="_blank">Will Zhang</a><br>Project memebrs: Ruby, Alexander, Cheng-Zhen, Daniel, Matthew,&amp; Lance</p>
```

Comment:
Typo in this line. "Neew Zealand", otherwise LGTM :)

Reply:
Yup thankk you Lance! Fixed now